src/modules/geospatial/geospatial.controller.ts

```diff
@@ -0,0 +1,29 @@
+import { Controller, Get, Query } from '@nestjs/common';
+import { ApiNotFoundResponse, ApiOperation, ApiResponse, ApiTags } from '@nestjs/swagger';
+
+import { GetAddressesByPostcodeQueryDto } from './dto/get-addresses-by-postcode-query.dto';
+import { GetAddressesResponse, GetAddressesResponseItem } from './dto/get-addresses-response.dto';
+import { GeospatialService } from './geospatial.service';
+
+@ApiTags('geospatial')
+@Controller('geospatial')
+export class GeospatialController {
+  constructor(private readonly geospatialService: GeospatialService) {}
+
+  @Get('addresses/postcode')
+  @ApiOperation({
+    summary:
+      "A search based on a property's postcode. Will accept a full valid postcode. Returns addresses from Ordnance Survey Delivery Point Address (DPA) system.",
+  })
+  @ApiResponse({
+    status: 200,
```

Comment:
Can the HTTP status codes please be reffered from `import { HttpStatusCodes } from 'axios` and then for 200 `HttpStatusCodes.Ok`.

Reply:
Changed to constant HttpStatus.OK in this controller, but left numeric codes in tests.